README.md

```diff
@@ -63,15 +63,46 @@ Depending on the RPC provider you are using, you may wish to alter `zkevm.rpc-ra
 
 ***
 
-## Running zKEVM Erigon
+## Running CDK-Erigon
 - Build using  `make cdk-erigon`
-- Set up your config file (copy an example and edit as required)
+- Set up your config file (copy one of the examples found in the repository root directory, and edit as required)
 - run `./build/bin/cdk-erigon --config="./hermezconfig-{network}.yaml"` (complete the name of your config file as required)
 
 NB: `--externalcl` flag is removed in upstream erigon so beware of re-using commands/config
 
+### Docker ([DockerHub](https://hub.docker.com/r/hermeznetwork/cdk-erigon))
+The image comes with 3 preinstalled default configs which you may wish to edit according to the config section below, otherwise you can mount your own config to the container as necessary.
+
+A datadir must be mounted to the container to persist the chain data between runs.
+
+Example commands:
+- Mainnet `docker run -p 8545:8545 -v  /datadirs:/datadirs hermeznetwork/cdk-erigon  --config="./mainnet.yaml" --datadir='/datadirs/mainnet'`
```

Comment:
What about using something like this that could be generic for all configurations?

`-v ./cdk-erigon-data:~/.cdk-erigon/data`

```
docker run -p 8545:8545 -v ./cdk-erigon-data/:~/.cdk-erigon/data hermeznetwork/cdk-erigon --config="./mainnet.yaml"
```
so the only change would be in the config file